use aoc::y2023::a_long_walk_23::do_solve;
use std::io::Error;

fn main() -> Result<(), Error> {
    aoc::with_input(2023, 23, do_solve)
}
